tests/suspend.rs

```diff
@@ -0,0 +1,115 @@
+//! Test suspension and un-suspension logic.
+
+use std::fs;
+
+use krill::{
+    commons::api::{ChildHandle, Handle, ResourceSet},
+    test::*,
+};
+
+#[tokio::test]
+async fn test_suspension() {
+    init_logging();
+
+    info("##################################################################");
+    info("#                                                                #");
+    info("#  Test that suspension and un-suspension of child CAs works.    #");
+    info("#                                                                #");
+    info("#   Uses the following lay-out:                                  #");
+    info("#                                                                #");
+    info("#                  TA                                            #");
+    info("#                   |                                            #");
+    info("#                testbed                                         #");
+    info("#                   |                                            #");
+    info("#                  CA                                            #");
+    info("#                                                                #");
+    info("#  - Disable the normal background jobs for this test            #");
+    info("#  - Set suspension threshold to 5 seconds                       #");
+    info("#  - Test that a certificate is published for CA                 #");
+    info("#  - Wait 6 seconds                                              #");
```

Comment:
This says wait 6 seconds but below the code waits 5 seconds.

Reply:
using code comments now